### PR TITLE
Mkdocs

### PR DESCRIPTION
**Pull request by SoulKyu**
Add mkdocs documentation + translation to gangway
@ultrabug could you please made a review on it ? :)

### CODE REVIEW

File: docs/index.md
Comment (ultrabug) [inline docs/index.md:1]: This file should be written in english

File: docs/index.fr.md
Comment (ultrabug) [inline docs/index.fr.md:1]: This file 'noun' description is in english, should be in french

File: docs/how-it-work/how-it-work.md
Comment (ultrabug) [inline docs/how-it-work/how-it-work.md:1]: rename the file to 'how-it-works' with a ending S ;) and the folder too

File: docs/how-it-work/custom-templates.md
Comment (ultrabug) [inline docs/how-it-work/custom-templates.md:1]: This file should be written in english

File: docs/getting-started/helm-deployment.md
Comment (ultrabug) [inline docs/getting-started/helm-deployment.md:1]: This file should be written in english

File: docs/getting-started/deployment.md
Comment (ultrabug) [inline docs/getting-started/deployment.md:1]: this file should be written in english

**Review by ultrabug — COMMENTED** [review]
Thanks a lot, some translations are missing, maybe that's on purpose and still WIP?

Site looks pretty good tho :+1:

File: docs/how-it-work/how-it-work.md
Comment (SoulKyu) [reply docs/how-it-work/how-it-work.md:1]: Done :)

File: docs/index.fr.md
Comment (SoulKyu) [reply docs/index.fr.md:1]: I think the translation made more sense in english.
are you sure that you want to traduce it ?

File: docs/how-it-work/custom-templates.md
Comment (SoulKyu) [reply docs/how-it-work/custom-templates.md:1]: Done

File: docs/getting-started/deployment.md
Comment (SoulKyu) [reply docs/getting-started/deployment.md:1]: Done

File: docs/getting-started/helm-deployment.md
Comment (SoulKyu) [reply docs/getting-started/helm-deployment.md:1]: Done :)

**Comment by ultrabug** [comment]
@SoulKyu I allowed myself to push several commits with fixes and improvements.

Please check them one by one!

**Review by ultrabug — CHANGES_REQUESTED** [review]
The last change I'm requesting is to update the first screenshot of the home page because it's still written gangly in the header!

**Comment by SoulKyu** [comment]
> The last change I'm requesting is to update the first screenshot of the home page because it's still written gangly in the header!

Yes working on it ! 

Thanks for your improvements

**Comment by SoulKyu** [comment]
It should be good now @ultrabug,
Thanks for the improvement specially on the UX :)

**Review by ultrabug — APPROVED** [review]

**Pull request merged**